elife/python.sls

```diff
@@ -35,7 +35,9 @@ python-dev:
 
 global-python-requisites:
     cmd.run:
-        - name: python3 -m pip install "virtualenv>=13"
```

Comment:
I imagine the constraint on 13 as a minimum is not necessary anymore?

Reply:
Almost definitely not. It would have been put there to force an upgrade at a guess. And it's probably been there since 14.04 days at least.